xdem/spatialstats.py

```diff
@@ -280,10 +282,233 @@ def interp_nd_binning(df: pd.DataFrame, list_var_names: Union[str,list[str]], st
     return interp_fun
 
 
+def two_step_standardization(dvalues: np.ndarray, list_var: Iterable[np.ndarray],
+                unscaled_error_fun: Callable[[tuple[np.ndarray, ...]], np.ndarray],
+                spread_statistic: Callable = nmad,
+                fac_spread_outliers: float | None = 7
+                ) -> tuple[np.ndarray, Callable[[tuple[np.ndarray, ...]], np.ndarray]]:
+    """
+    Standardize the proxy differenced values using the modelled heteroscedasticity, re-scaled to the spread statistic,
+    and generate the final standardization function.
+
+    :param dvalues: Proxy values array of size (N,)
```

Comment:
I'm not sure the term "proxy" will be clear enough for everyone. Maybe specify "Values where signal (e.g. elevation change) is expected to be 0, to be used as proxy for error"?

Reply:
Changed everywhere into "Proxy values as array or Raster (i.e., differenced values where signal should be zero such as elevation differences on stable terrain)"